apps/template/index.html

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/svg+xml" href="src/assets/favicon.ico" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>React 18, TypeScript, Vitest, Testing Library, TailwindCSS 3, Vite, Eslint, Juno UI components and Prettier boilerplate</title>
+  </head>
+  <body>
+    <div id="root"></div>
+    <script type="module" src="/src/index.tsx"></script>
```

Comment:
I'd like to demonstrate how we pass appProps to the application, ensuring that this template is pre-configured when used for any future application. Please see the example App or doop:
https://github.com/cloudoperators/juno/blob/main/apps/example/index.html#L34

Reply:
Why do we need that in a first place?